### ISSUE TITLE
MD029 Ordered list item prefix is error and bug

### ISSUE THREAD

**Issue by timmy61109** [DavidAnson/markdownlint#998]
the test is error, so my CI not pass. but I don't why.

```markdown
## 團隊增進

1. 範本複製：自動將 `example` 中的自動化複製到各個專案中的範本
2. 團隊報告：使用爬蟲將資料收集，製作團隊成效、效率、效能的統計與數據報告
3. 議題產生器：將活動標準流程，透過程式自動產生議題
4. 環境佈署：針對 Windows 進行環境佈署，可以使用 Python 來自動設定與處理
```

```bash
README.md:10: MD029 Ordered list item prefix
README.md:11: MD029 Ordered list item prefix
README.md:12: MD029 Ordered list item prefix
README.md:19: MD029 Ordered list item prefix
README.md:18: MD029 Ordered list item prefix
README.md:25: MD029 Ordered list item prefix
README.md:26: MD029 Ordered list item prefix
README.md:31: MD029 Ordered list item prefix
README.md:32: MD029 Ordered list item prefix
```

I use pymarkdown is pass.

**Comment by DavidAnson** [comment]
The example you show does not produce any violations with this library as you can see with the link below. If you are using a different tool or have a different configuration, please provide complete steps to reproduce what you are seeing.

https://dlaa.me/markdownlint/#%25m%23%20Issue%20998%0A%0A%23%23%20%E5%9C%98%E9%9A%8A%E5%A2%9E%E9%80%B2%0A%0A1.%20%E7%AF%84%E6%9C%AC%E8%A4%87%E8%A3%BD%EF%BC%9A%E8%87%AA%E5%8B%95%E5%B0%87%20%60example%60%20%E4%B8%AD%E7%9A%84%E8%87%AA%E5%8B%95%E5%8C%96%E8%A4%87%E8%A3%BD%E5%88%B0%E5%90%84%E5%80%8B%E5%B0%88%E6%A1%88%E4%B8%AD%E7%9A%84%E7%AF%84%E6%9C%AC%0A2.%20%E5%9C%98%E9%9A%8A%E5%A0%B1%E5%91%8A%EF%BC%9A%E4%BD%BF%E7%94%A8%E7%88%AC%E8%9F%B2%E5%B0%87%E8%B3%87%E6%96%99%E6%94%B6%E9%9B%86%EF%BC%8C%E8%A3%BD%E4%BD%9C%E5%9C%98%E9%9A%8A%E6%88%90%E6%95%88%E3%80%81%E6%95%88%E7%8E%87%E3%80%81%E6%95%88%E8%83%BD%E7%9A%84%E7%B5%B1%E8%A8%88%E8%88%87%E6%95%B8%E6%93%9A%E5%A0%B1%E5%91%8A%0A3.%20%E8%AD%B0%E9%A1%8C%E7%94%A2%E7%94%9F%E5%99%A8%EF%BC%9A%E5%B0%87%E6%B4%BB%E5%8B%95%E6%A8%99%E6%BA%96%E6%B5%81%E7%A8%8B%EF%BC%8C%E9%80%8F%E9%81%8E%E7%A8%8B%E5%BC%8F%E8%87%AA%E5%8B%95%E7%94%A2%E7%94%9F%E8%AD%B0%E9%A1%8C%0A4.%20%E7%92%B0%E5%A2%83%E4%BD%88%E7%BD%B2%EF%BC%9A%E9%87%9D%E5%B0%8D%20Windows%20%E9%80%B2%E8%A1%8C%E7%92%B0%E5%A2%83%E4%BD%88%E7%BD%B2%EF%BC%8C%E5%8F%AF%E4%BB%A5%E4%BD%BF%E7%94%A8%20Python%20%E4%BE%86%E8%87%AA%E5%8B%95%E8%A8%AD%E5%AE%9A%E8%88%87%E8%99%95%E7%90%86%0A

**Comment by DavidAnson** [comment]
Based on the formatting of the output you show, I think you are using the Ruby implementation which is a separate project you can find and open issues with here: https://github.com/markdownlint/markdownlint

**Comment by timmy61109** [comment]
> Based on the formatting of the output you show, I think you are using the Ruby implementation which is a separate project you can find and open issues with here: https://github.com/markdownlint/markdownlint

Yes, I use Ruby, so I don't why.

**Comment by timmy61109** [comment]
> The example you show does not produce any violations with this library as you can see with the link below. If you are using a different tool or have a different configuration, please provide complete steps to reproduce what you are seeing.
> 
> https://dlaa.me/markdownlint/#%25m%23%20Issue%20998%0A%0A%23%23%20%E5%9C%98%E9%9A%8A%E5%A2%9E%E9%80%B2%0A%0A1.%20%E7%AF%84%E6%9C%AC%E8%A4%87%E8%A3%BD%EF%BC%9A%E8%87%AA%E5%8B%95%E5%B0%87%20%60example%60%20%E4%B8%AD%E7%9A%84%E8%87%AA%E5%8B%95%E5%8C%96%E8%A4%87%E8%A3%BD%E5%88%B0%E5%90%84%E5%80%8B%E5%B0%88%E6%A1%88%E4%B8%AD%E7%9A%84%E7%AF%84%E6%9C%AC%0A2.%20%E5%9C%98%E9%9A%8A%E5%A0%B1%E5%91%8A%EF%BC%9A%E4%BD%BF%E7%94%A8%E7%88%AC%E8%9F%B2%E5%B0%87%E8%B3%87%E6%96%99%E6%94%B6%E9%9B%86%EF%BC%8C%E8%A3%BD%E4%BD%9C%E5%9C%98%E9%9A%8A%E6%88%90%E6%95%88%E3%80%81%E6%95%88%E7%8E%87%E3%80%81%E6%95%88%E8%83%BD%E7%9A%84%E7%B5%B1%E8%A8%88%E8%88%87%E6%95%B8%E6%93%9A%E5%A0%B1%E5%91%8A%0A3.%20%E8%AD%B0%E9%A1%8C%E7%94%A2%E7%94%9F%E5%99%A8%EF%BC%9A%E5%B0%87%E6%B4%BB%E5%8B%95%E6%A8%99%E6%BA%96%E6%B5%81%E7%A8%8B%EF%BC%8C%E9%80%8F%E9%81%8E%E7%A8%8B%E5%BC%8F%E8%87%AA%E5%8B%95%E7%94%A2%E7%94%9F%E8%AD%B0%E9%A1%8C%0A4.%20%E7%92%B0%E5%A2%83%E4%BD%88%E7%BD%B2%EF%BC%9A%E9%87%9D%E5%B0%8D%20Windows%20%E9%80%B2%E8%A1%8C%E7%92%B0%E5%A2%83%E4%BD%88%E7%BD%B2%EF%BC%8C%E5%8F%AF%E4%BB%A5%E4%BD%BF%E7%94%A8%20Python%20%E4%BE%86%E8%87%AA%E5%8B%95%E8%A8%AD%E5%AE%9A%E8%88%87%E8%99%95%E7%90%86%0A

I use docker install, use ruby install, and in run, I use GitLab Runner run CI, but I use my computer same.

**Comment by timmy61109** [comment]
```yml
 Markdown lint tool:
   stage: test
   script:
     - gem install mdl
     - mdl .
     - mdl README.md docs
```

**Comment by DavidAnson** [comment]
This is the wrong place to report issues with the Ruby implementation of mdl that you are using. Please go here instead: https://github.com/markdownlint/markdownlint

**Comment by timmy61109** [comment]
> This is the wrong place to report issues with the Ruby implementation of mdl that you are using. Please go here instead: https://github.com/markdownlint/markdownlint

Ok, thank you
